packages/react/src/components/Tile/Tile-test.js

```diff
@@ -214,26 +213,27 @@ describe('Tile', () => {
       expect(wrapper.state().expanded).toEqual(true);
     });
 
-    it('displays the default tooltip for the chevron depending on state', () => {
+    it('displays the default tooltip for the button depending on state', () => {
       const defaultExpandedIconText = 'Collapse';
       const defaultCollapsedIconText = 'Expand';
 
       // Force the expanded tile to be collapsed.
       wrapper.setState({ expanded: false });
-      const collapsedDescription = wrapper.find(ChevronDown16).getElements()[0]
+      const collapsedDescription = wrapper.find('button').getElements()[0]
```

Comment:
Would it help to use 

```
wrapper.find(`[aria-label="${defaultExpandedIconText}"]`)
```

Just so it's not tied to a specific node and instead is tied to the attribute. The assertion after becomes that this node exists instead of checking.

Another idea is just `wrapper.find('[aria-label]')` to get the node itself